middleware/faq.rst

```diff
@@ -535,6 +535,251 @@ The resulting query logs can be useful for developers and database administrator
+           forcedPhotCoadd       0        216      78      0     0   294      294
+          writeObjectTable       0         36      13      0     0    49       49
+      transformObjectTable       0         36      13      0     0    49       49
+    consolidateObjectTable       0          0       1      0     0     1        1
+        matchObjectToTruth       0          0       1      0     0     1        1
+      compareObjectToTruth       0          0       1      0     0     1        1
+
+
+which indicates that there are 6 failed ``templateGen`` quanta, 14 failed ``assembleCoadd`` quanta, 14 blocked ``detection`` quanta, and so on.
+
+These totals are all reached by examining the status of each task run on each data ID from run to run.
+
+Statuses for Quanta 
+^^^^^^^^^^^^^^^^^^^
+
+Unknown
+"""""""
+The ``Unknown`` category could mean one of two things. Being the default value for the status of a task run on a particular data ID, it could technically mean that no attempt has been made to execute this part of the graph. However, in a table that does not look empty, it much more likely denotes that the task's metadata is missing for that data ID. This makes it impossible for us to tell whether the task succeeded or failed, but does point to possible infrastructure problems.
+
+Successful
+""""""""""
+This status is the trademark of a successful quantum. The specific path to being marked as successful by the `QuantumProvenanceGraph` is that metadata and log datasets exist for the task for the data ID in question.
+
+Blocked
+"""""""
+Blocked quanta are the successors of unsuccesful quanta which cannot be executed because their inputs are not present. They do not represent failure per se, but since they could not execute, they also do not produce output data products. The way that the `QuantumProvenanceGraph` identifies a ``Blocked`` quantum is that it has no metadata and no logs (so it did not start or finish execution) **and** that the task run on that particular data ID is a successor (according to the quantum graph) of a ``Failed`` quantum.
+
+Failed
+""""""
+There are log datasets, but no metadata datasets, for this task and data ID, indicating that the task started but did not finish execution.
+
+
+Wonky
+"""""
+A Wonky quantum is the result of infrastructure problems or concerning middleware mismatches. The category is intended to halt processing and require human intervention to proceed. As such, when a task is marked as ``Wonky`` for a particular data ID, no further successes (or any other statuses, for that matter) will change the overall status out of ``Wonky``. A quantum can only exit a ``Wonky`` state via human intervention. Currently, there are three paths to the ``Wonky`` state for a quantum:
+
+- A quantum which was marked as ``Successful`` on a previous processing attempt (run) has a more recent attempt (run) which the `QuantumProvenanceGraph` identifies as unsuccessful
+
```

Comment:
The empty line here will place the following sentence into its own paragraph. I'd advise to move this parenthesis block of text onto the end of the above sentence, i.e.:

> ... identifies as unsuccessful (i.e., Graph 1...